File: swift/6.swift
import Foundation

func sumOfSquares(max: Int) -> Int {
  var sum = 0

  for i in 1...max {
    sum += Int(pow(Double(i), 2))
  }

  return sum
}

func squareOfSum(max: Int) -> Int {
  var sum = 0

  for i in 1...max {
    sum += i
  }

  return Int(pow(Double(sum), 2))
}

print(abs(squareOfSum(max: 100) - sumOfSquares(max: 100)))
// 25_164_150

File: julia/6.jl
function sumOfSquares()
	sum = 0
	for i = 1:100
		sum += i^2
	end

	return sum
end

function squareOfSum()
	sum = 0
	for i = 1:100
		sum += i
	end

	return sum^2
end

println(squareOfSum() - sumOfSquares())
# 25164150
